Shell scripting bug: 'continue' statements in PROCESS_ALL_METADATA_PREFIXES cause HTTP 422 errors and script termination

## Problem Description
The `setup` command in `Makefile.ghprj` intermittently fails with HTTP 422 validation errors during GitHub label creation, despite individual API calls working correctly.

## Root Cause Analysis (5-Why Technique)

**Problem**: HTTP 422 validation errors during GitHub label setup

1. **Why does the script fail with HTTP 422 validation errors?**
   - GitHub API rejects some requests during bulk label creation

2. **Why do individual API calls work but the script fails?**
   - Individual manual API calls succeed (CREATE/UPDATE both work)
   - The issue is in script execution flow, not API operations

3. **Why does the script fail even when labels exist?**
   - The script attempts to process labels but terminates prematurely
   - Error occurs in the while loop processing `/tmp/workflow_specs.txt`

4. **Why does the while loop terminate prematurely?**
   - `continue` statements in `PROCESS_ALL_METADATA_PREFIXES` function cause unexpected behavior

5. **Why do `continue` statements cause script termination?**
   - **ROOT CAUSE**: Shell scripting bug with `continue` statements inside `define` blocks executed within while loops under `-euo pipefail` shell flags

## Technical Details

### The Bug
In `PROCESS_ALL_METADATA_PREFIXES` function, when a prefix is empty (e.g., `CONFIDENCE_PREFIX=""`), the script executes:

```bash
if [ -n "$$CONFIDENCE_PREFIX" ]; then \
  name="$$CONFIDENCE_PREFIX/$$name"; \
else \
  continue; \  # <-- BUG: This breaks script execution
fi; \
```

### Shell Configuration
The Makefile uses strict shell settings:
```makefile
SHELL := /bin/bash
.ONESHELL:
.SHELLFLAGS := -euo pipefail -c
```

With `-euo pipefail`, the `continue` statement inside a `define` block called within a while loop causes unexpected script termination.

## Reproduction Steps

1. Run the setup command:
   ```bash
   curl -fsSL https://raw.githubusercontent.com/tbrandenburg/ghaw/main/Makefile.ghprj | make -f - setup
   ```

2. Observe the failure pattern:
   ```
   UPDATE  priority     /medium
   gh: Validation Failed (HTTP 422)
   make[1]: *** [/tmp/GmV1B3df:561: _setup] Error 1
   ```

3. Note that individual API calls work perfectly:
   ```bash
   # This succeeds
   gh api -X POST "repos/owner/repo/labels" -f name="priority     /low" -f color="28A745" -f description="Low priority"
   ```

## Evidence
- Manual API calls (CREATE/UPDATE) succeed with identical parameters
- URL encoding is correct: `priority%20%20%20%20%20%2Flow`
- GitHub CLI authentication and permissions work fine
- Error occurs consistently at the same point in execution
- Dry-run mode works because it doesn't execute the problematic API calls

## Proposed Solution

Replace `continue` statements with conditional processing:

```bash
define PROCESS_ALL_METADATA_PREFIXES
if echo "$$name" | grep -q "^priority:"; then \
  name="$${name#priority:}"; \
  if [ -n "$$PRIORITY_PREFIX" ]; then \
    name="$$PRIORITY_PREFIX/$$name"; \
  fi; \
elif echo "$$name" | grep -q "^severity:"; then \
  name="$${name#severity:}"; \
  if [ -n "$$SEVERITY_PREFIX" ]; then \
    name="$$SEVERITY_PREFIX/$$name"; \
  fi; \
# ... continue for other prefixes without using 'continue'
else \
  if [ -n "$$PREFIX" ]; then \
    name="$$PREFIX/$$name"; \
  fi; \
fi
endef
```

Or use a flag-based approach to skip processing instead of `continue`.

## Impact
- **Severity**: High - Setup command fails intermittently
- **Workaround**: Run setup multiple times or create labels manually
- **Fix Complexity**: Low - Simple shell scripting correction needed

## Environment
- OS: Linux
- Shell: bash 5.x
- GitHub CLI: 2.45.0
- Make: GNU Make 4.x

## Additional Notes
This affects repositories where some metadata label prefixes are intentionally disabled (set to empty strings), which is a supported configuration according to the documentation.

--- a/Makefile.ghprj
+++ b/Makefile.ghprj
@@ -383,36 +383,26 @@ if echo "$$name" | grep -q "^priority:"; then \
   name="$${name#priority:}"; \
   if [ -n "$$PRIORITY_PREFIX" ]; then \
     name="$$PRIORITY_PREFIX/$$name"; \
-  else \
-    continue; \
   fi; \
 elif echo "$$name" | grep -q "^severity:"; then \
   name="$${name#severity:}"; \
   if [ -n "$$SEVERITY_PREFIX" ]; then \
     name="$$SEVERITY_PREFIX/$$name"; \
-  else \
-    continue; \
   fi; \
 elif echo "$$name" | grep -q "^confidence:"; then \
   name="$${name#confidence:}"; \
   if [ -n "$$CONFIDENCE_PREFIX" ]; then \
     name="$$CONFIDENCE_PREFIX/$$name"; \
-  else \
-    continue; \
   fi; \
 elif echo "$$name" | grep -q "^complexity:"; then \
   name="$${name#complexity:}"; \
   if [ -n "$$COMPLEXITY_PREFIX" ]; then \
     name="$$COMPLEXITY_PREFIX/$$name"; \
-  else \
-    continue; \
   fi; \
 elif echo "$$name" | grep -q "^rank:"; then \
   name="$${name#rank:}"; \
   if [ -n "$$RANK_PREFIX" ]; then \
     name="$$RANK_PREFIX/$$name"; \
-  else \
-    continue; \
   fi; \
 else \
   if [ -n "$$PREFIX" ]; then \

